lib/core/buttons-menu/buttons-menu.component.stories.ts

```diff
@@ -0,0 +1,103 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { CoreStoryModule } from '../testing/core.story.module';
+import { ButtonsMenuComponent } from './buttons-menu.component';
+import { ButtonsMenuModule } from './buttons-menu.module';
+import { MatIconModule } from '@angular/material/icon';
+
+export default {
+    component: ButtonsMenuComponent,
+    title: 'Core/Components/ButtonsMenu',
+    decorators: [
+        moduleMetadata({
+            imports: [CoreStoryModule, ButtonsMenuModule, MatIconModule]
```

Comment:
please move MatIconModule to ButtonsMenuModule and remove MaterialModule. we should use only modules that we need
https://alfresco.atlassian.net/jira/software/c/projects/HXENG/boards/868?modal=detail&selectedIssue=HXENG-74